### PR TITLE
Adapter for Stockholm, Sweden

### PR DESCRIPTION
**Pull request by lhufnagel**
Hey,
I wrote an adapter for the air measurements in the Stockholm area (Sweden).
They are extracted from http://slb.nu/slbanalys/luften-idag/ an unit of the Environmental Administration in Stockholm. Those measurements are unfortunately not part of the EEA dataset

### CODE REVIEW

**Comment by jflasher** [comment]
Awesome, thanks @lhufnagel! I'll try to take a look at this soon. Travis is complaining about some syntax errors though https://travis-ci.org/openaq/openaq-fetch/builds/192650177#L1293

File: adapters/sweden.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    // Parse data table
+    var rows = node.split('],[');
+    var legend = rows.shift().split(',');
+
+    // Iterate over the last 24 hours of measurements
+    rows.forEach((row, index) => {
+      row = row.split(']').join('').trim().split(',');
+
+      // First column contains the hour of the recordings
+      var date = moment(row[0], 'HH:mm').date(moment().date());
```
Comment (jflasher) [inline adapters/sweden.js]: Just did a quick spot check and time doesn't seem to be matching up. The adapter is giving me 

```json
{"date":{"utc":"2017-01-19T00:00:00.000Z","local":"2017-01-19T01:00:00+01:00"},"parameter":"no2","location":"Hornsgatan","value":43.76,"unit":"µg/m³","city":"Stockholm","attribution":[{"name":"SLB","url":"http://slb.nu/slbanalys/luften-idag/"}],"averagingPeriod":{"value":1,"unit":"hours"},"coordinates":{"latitude":59.31713214,"longitude":18.04878744},"country":"SE","sourceName":"Sweden","sourceType":"government","mobile":false}
```

but the page is showing that value at 19:00. Noticed this for other measurements as well.

File: adapters/sweden.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    }
+  });
+};
+
+var formatData = function (result) {
+  var measurements = [];
+
+  var getCoordinates = function (id) {
+    switch (id) {
+      case 'Högdalen':
```
Comment (jflasher) [inline adapters/sweden.js]: Could you make a note in here where these coordinates come from? Ideally we'd be getting them programmatically somehow, but that's not always possible.

File: adapters/sweden.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      rendered = rendered.replace('[', '').trim();
+      rendered = rendered.split("'").join('').trim(); // remove all [
+
+      if (rendered.length > 20) {
+        nodes.push(rendered);
+      }
+    }
+  });
+
+  var parameters = ['pm10', 'no2', 'pm25', 'o3']; // currently this order
```
Comment (jflasher) [inline adapters/sweden.js]: Any way to get these without hard coding them?

File: adapters/sweden.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+  // Iterate over different quantitites (PM10, NO2, PM25, O3)
+  nodes.forEach((node, nodeIndex) => {
+    // Parse data table
+    var rows = node.split('],[');
```
Comment (jflasher) [inline adapters/sweden.js]: Love it!

File: sources/se.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        "url": "http://slb.nu/slbanalys/luften-idag/",
+        "adapter": "sweden",
+        "name": "Sweden",
+        "city": "Stockkholm",
+        "country": "SE",
+        "description": "Swedish data from SLB analys",
+        "sourceURL": "http://slb.nu/slbanalys/luften-idag/",
+        "contacts": [
+            "info@openaq.org"
```
Comment (jflasher) [inline sources/se.json]: This is currently unused, but may be used to let people know if there are adapter errors in the future. Feel free to put your email in here if you want.

**Comment by jflasher** [comment]
Thanks for the updates. I'm out for a week but will check it out once I'm back.

**Comment by RocketD0g** [comment]
Awesome work, @lhufnagel! Thanks so much! :1st_place_medal:

**Pull request closed without merging**